### PR TITLE
[5.0] Add disk container symlink

### PR DESCRIPTION
**Pull request by maximenoel8**
## What does this PR change?
Use symbolic link to add extra disks to container

I followed this issue https://github.com/SUSE/spacewalk/issues/23341 for the folder names. We can change them if you prefer something else.
About the user / group configuration, I checked the default folder rights used for the container and they are own by root. That's why I removed the user configuration but fill free to correct me if I'm wrong.

I also wander if I could not skip the `spacewalk_data_directory` and directly create `/var/lib/containers/storage/volumes/var-spacewalk` in `spacewalk_directory`

If we accept this PR I will close : https://github.com/uyuni-project/sumaform/pull/1474
Fixes https://github.com/SUSE/spacewalk/issues/23516

### CODE REVIEW

File: modules/server_containerized/main.tf [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 variable "images" {
   default = {
-    "head"           = "sles15sp4o"
+    "head"           = "slemicro55-ign"
```
Comment (srbarrios) [inline modules/server_containerized/main.tf:5]: True, but we can't change this directly, we first need to align with another PR we are working on.
Let's keep this change out of this PR please.

**Review by srbarrios — APPROVED** [review]
Approving, but please read by comment about head image before merging.

**Review by srbarrios — CHANGES_REQUESTED** [review]
It seems something is wrong, check the GH checks:
https://github.com/uyuni-project/sumaform/actions/runs/7663252752/job/20885774161?pr=1480#step:9:134

File: modules/server_containerized/main.tf (hunk 1 above)
Comment (cbosdo) [reply modules/server_containerized/main.tf:5]: I'll be fixing this in my PR for sure

**Review by cbosdo — COMMENTED** [review]
I still don't get why you need those two disks and not just one, but the PR looks good to me otherwise.

**Review by rjmateus — APPROVED** [review]
looks good.

**Comment by maximenoel8** [comment]
@srbarrios Could you review this PR and see if the changes are correct please

**Review by nodeg — COMMENTED** [review]
Maxime, you should also take a look at the test and adjust the testing template by adding the 2 missing variables.

```bash
╷
│ Error: Reference to undeclared input variable
│ 
│   on modules/server_containerized/main.tf line 35, in module "server_containerized":
│   35:   additional_disk_size          = var.additional_disk_size
│ 
│ An input variable with the name "additional_disk_size" has not been
│ declared. This variable can be declared with a variable
│ "additional_disk_size" {} block.
╵
╷
│ Error: Reference to undeclared input variable
│ 
│   on modules/server_containerized/main.tf line 36, in module "server_containerized":
│   36:   second_additional_disk_size   = var.second_additional_disk_size
│ 
│ An input variable with the name "second_additional_disk_size" has not been
│ declared. This variable can be declared with a variable
│ "second_additional_disk_size" {} block.
```

**Comment by maximenoel8** [comment]
Thanks, probably a rebase change ...

**Review by srbarrios — APPROVED** [review]

**Pull request merged**